Tao_Current.txt

```diff
@@ -1,73 +1,84 @@
+| I-D    | Internet-Draft                                      |
+| IESG   | Internet Engineering Steering Group                 |
+| IETF   | Internet Engineering Task Force                     |
+| IPR    | Intellectual property rights                        |
+| IRTF   | Internet Research Task Force                        |
+| ISOC   | Internet Society                                    |
+| RFC    | Request for Comments                                |
+| STD    | Standard (RFC)                                      |
+| WG     | Working Group                                       |
+
+
 2. What Is the IETF?
 
 The IETF is a loosely self-organized group of people who contribute to the engineering and evolution of Internet technologies. It is the principal body engaged in the development of new Internet standard specifications. The IETF is unusual in that it exists as a collection of happenings, but is not a corporation and has no board of directors, no members, and no dues; see [RFC 3935], "A Mission Statement for the IETF", for more detail.
 
 Its mission includes the following:
 
-    Identifying, and proposing solutions to, pressing operational and technical problems in the Internet
-    Specifying the development or usage of protocols and the near-term architecture to solve such technical problems for the Internet
-    Making recommendations to the Internet Engineering Steering Group (IESG) regarding the standardization of protocols and protocol usage in the Internet
-    Facilitating technology transfer from the Internet Research Task Force (IRTF) to the wider Internet community
-    Providing a forum for the exchange of information within the Internet community between vendors, users, researchers, agency contractors, and network managers
+    - Identifying and proposing solutions to pressing operational and technical problems in the Internet
+    - Specifying the development or usage of protocols and the near-term architecture to solve such technical problems for the Internet
+    - Making recommendations to the Internet Engineering Steering Group (IESG) regarding the standardization of protocols and protocol usage in the Internet
+    - Facilitating technology transfer from the Internet Research Task Force (IRTF) to the wider Internet community
+    - Providing a forum for the exchange of information within the Internet community between vendors, users, researchers, agency contractors, operators, and network managers
+
+The IETF mission further states that the Internet isn't value-neutral, and neither is the IETF.  We want the Internet to be useful for communities that share our commitment to openness and fairness. We embrace technical concepts such as decentralized control, edge-user empowerment and sharing of resources, because those concepts resonate with the core values of the IETF community.  These concepts have little to do with the technology that's possible, and much to do with the technology that we choose to create.
```

Comment:
What does "sharing of resources" mean here?

Reply:
That text comes straight from BCP95